### DISCUSSION TITLE
JSON Schema 模式下如何在校验时触发自定义组件内部的校验方法

### DISCUSSION THREAD

**Discussion by chasingsnail** [alibaba/formily#3942] [Q&A 2.x]
在 JSON Schema 模式中使用了自定义组件，想要在提交的时候，在自定义校验方法中触发组件内部的校验函数。请问要如何实现？
目前的思路是想获取到组件的实例，然后触发，但是好像没有找到相关的方法

**Comment by miaoziwei** [comment]
我也是在JSON Schema 里使用了自定义组件，然后在表单提交的时候获取不到自定义组件的input框输入的值，是用connect创建的一个组件，请问您知道怎么解决吗？
